### ISSUE TITLE
Create Practice Tracks for "Down in the River to Pray"

### ISSUE THREAD

**Issue by Jim-Duke** [Jim-Duke/concordia-sacrae#1]
Create the practice tracks for the song "Down in the River to Pray".  A copy of the PDF has been uploaded to the Concordia Sacrae shared folder.  It can be found here: https://1drv.ms/b/s!Anuauc_9g6e1gts5WwW3fJghYW9z0A?e=owJrtW

**Comment by nocollier** [comment]
I wonder what your opinion is on the following:

* My inclination is to treat the alto and tenor solo/duet as just part of the alto1 and tenor1 part and include rests in all other parts. Is this how you would like to encode this?
* There are splits in a good section of the alto and tenor parts. I could just render them as chords, but that may confuse people. Otherwise I could split alto into alto 1 and alto 2 and the same for tenors. Should I then create a `saattb` set of templates?
* There is maybe a single measure of split soprano and bass, do we split them into 8 parts just for this? Or render the midi with a chord?

**Comment by Jim-Duke** [comment]
- Treating the solo/duet as just parts of the alto1 and tenor1 parts is fine.  However, I have created solo voices before - even if the solo was only a few measures.  It has to do with how I perceive the practice tracks being used.  More on that later.
- So far as including rests in all other pats - yes, that's how I do it.  Sure, if I were rendering this for print publication, I might do it differently.  But we're not; and just including rests is far far easier.
- I tend to separate them into individual voices.  So, someone who thinks of themselves as an Alto I will listen to the alto1 tracks; and an Alto II will listen to the alto2 tracks.  And the track will follow the splits properly.  Besides - it disambiguates where the spits should go.  Most of the time it's straight forward.  But sometimes it's not (if alto and sopranos share the same staff, and you have a 3 way split - what do you do?).  In times past I would make my own judgment - but pass those questions by Scott to get his thoughts; then encode the voices to follow Scott's guidance.  And yes, that's why there are so many templates.  Or, you could go with ssaattbb and just leave one voice all rests and for the most part ignore it.
- I generally like to encode separate voices - to be clear where the different voices should go.  In "Nearer My God to Thee", most of the time the two voices that share a staff are unison - yet I made separate voices.

I try to make it VERY clear where the voice splits are.  So even if they're unison most of the time, it's nice to know that when I'm listening to the bass2 track, it follows what the Bass II's should do.  Even if it's only for a measure.  And with copy/paste it's not all that much work to do.

**Comment by Jim-Duke** [comment]
And where there are short duration solos - I tend to create a whole separate voice for them.  It also allows me to determine when the soloist should stop singing what the rest of their normal voice is singing (if that's an issue - I don't think that situation applies in the song you're working on).  But, let's say there is a small soprano solo in the middle of a song; but the rest of the sopranos keep singing through it.  It's nice to show where the soloist should stop singing with the rest of the sopranos, then sing their solo, then join back in.  I think daniel from the third season has that kind of thing; and My god is a rock, as well for a soprano soloist.

**Comment by Jim-Duke** [comment]
May need to revisit how we use this tool.  The issue isn't finished once the code is merged in.  We then put the product out for review, and it would be nice to have the issue track that as well.  Perhaps have a separate issue to accompany that phase?

**Comment by nocollier** [comment]
Hmm, good point. Would it work to just not link the Issues to the PR? The issue discussion is easy enough to locate on its own by the naming convention?

**Comment by Jim-Duke** [comment]
I think you’re correct.  It didn’t occur to me that when I merged the PR that it would close the linked issue.  That’s the easiest course.
